WcaOnRails/app/assets/stylesheets/competitions.scss

```diff
@@ -150,3 +182,67 @@ $venue-map-wrapper-height: 400px;
 .big-column {
   width: 100%;
 }
+
+.competition-select.form-inline {
+  background-color: #fff;
+  width: 100%;
+  padding-bottom: 10px;
+
+  input,
+  select {
+    font-size: 12px;
+    font-weight: normal;
+    display: block;
+  }
+
+  button {
+    width: 100px;
+  }
+
+  .btn-group {
+    display: block;
+  }
+}
+
+.same-line {
+  display: inline-block;
+  margin-right: 5px;
+}
+
+#today-line {
+  @extend .bg-primary;
+}
+
+#scroller {
+  width: 100%;
+}
+
+.competition-list {
+  li:nth-child(odd) {
+    background-color: $table-bg-accent;
+  }
+
+  .badge {
+    font-size: 14px;
+    color: #000;
+    background-color: unset;
+  }
+}
+
+.competition-info {
+  p {
+    margin: 0;
+  }
+
+  &.in-progress {
```

Comment:
Can you add a little comment here about why `.in-progress` is special? Will this look weird once `.in-progress` is *not* special, ie: when on mobile and in progress, ongoing, and upcoming all take up 12 columns?